contracts/hts-precompile/IHRC.sol

```diff
@@ -1,7 +1,15 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.7;
+pragma solidity >=0.4.9 <0.9.0;
+
+interface IERCCommonToken {
+    function balanceOf(address account) external view returns (uint256);
+}
 
 interface IHRC {
     function associate() external returns (uint256 responseCode);
     function dissociate() external returns (uint256 responseCode);
+
+    function isAssociated(address evmAddress) external view returns (bool); // TODO: pending completion of https://hips.hedera.com/hip/hip-719 ??
```

Comment:
Not implemented yet unfortunately

Reply:
no harm in keeping it in as it will eventually?

Reply:
The opposite actually, we share this file and others as the implemented feature set, adding it here gives the wrong impression of support for developers that may take the interface and consume it